core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/SequenceDiagramServices.java

```diff
@@ -140,70 +111,28 @@ public static EObject getSendingEnd(SequenceMessage message) {
     MessageEnd end = message.getSendingEnd();
     if (end == null) {
       return message; // found message case
-    }
+    } else {
```

Comment:
Could you remove the unnecessary else clause?

Reply:
In our Dev environment, without the "else", we have a Checsktyle warning.

Reply:
In our dev environment Sonar checks this kind of rule :)